java/fury-core/src/main/java/io/fury/Fury.java

```diff
@@ -1146,6 +1136,22 @@ private Object deserializeFromStream(
     }
   }
 
+  private static void transferObjectFromStream(InputStream inputStream, MemoryBuffer buffer) throws IOException {
```

Comment:
`readToBufferFromStream` seems more intuitive to me, since we are copy binary instead of object